server/form-pages/apply/risk-and-need-factors/location-factors/preferredAps.ts

```diff
@@ -12,43 +12,50 @@ export type PreferredApsBody = {
   preferredAp3: string
   preferredAp4: string
   preferredAp5: string
-  selectedAps: Array<ApprovedPremises>
+  selectedAps: Array<Cas1PremisesSummary>
 }
 
 const preferredAps = new Array(5).fill('').map((_, i) => `preferredAp${i + 1}`)
 
+type SelectorAttributes = {
+  'data-premises-with-areas'?: boolean
+  'data-region-prompt'?: string
+}
 @Page({
   name: 'preferred-aps',
   bodyProperties: [...preferredAps, 'selectedAps'],
 })
 export default class PreferredAps implements TasklistPage {
-  title = 'Select a preferred AP'
+  title: string
 
-  allPremises: Array<ApprovedPremises> = []
+  allPremises: Array<Cas1PremisesSummary> = []
 
   preferredApOptions = preferredAps
 
   preferredApLabels = this.preferredApOptions.map((_, i) => `${numberToOrdinal(i)} choice AP`)
 
+  isWomensAp: boolean
```

Comment:
Is this used anywhere? I can't find any direct usage, and it seems `application.isWomensApplication` should cover all bases? 

In any case, the abbreviation `ap` tends to refer to Approved Premises around this codebase, so to avoid confusion I'd recommend using the full `isWomensApplication` if this is still needed.